xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java

```diff
@@ -236,15 +236,15 @@ public void printVerbatimContent(String verbatimContent)
             }
         }
 
-        if (subVerbatimStack.size() > 0) {
+        if (!subVerbatimStack.isEmpty()) {
             // Append remaining string
             subVerbatimStack.peek().append(end);
 
             // Escape not closed verbatim blocks
-            while (subVerbatimStack.size() > 0) {
+            while (!subVerbatimStack.isEmpty()) {
```

Comment:
same here

Reply:
TBH I'm personally finding it easier to read "subVerbatimStack.size() > 0" than "!subVerbatimStack.isEmpty()" .... :) But let's consider it's not a false positive for now...